python/ql/src/semmle/python/RegexTreeView.qll

```diff
@@ -445,7 +445,7 @@ class RegExpEscape extends RegExpNormalChar {
     or
     this.getUnescaped() = "t" and result = "\t"
     or
-    this.getUnescaped() = "f" and result = "\f"
```

Comment:
I wonder if this is sustainable, or if we should elide this case until more escapes are supported by CodeQL...

Reply:
Elided for now..

Reply:
Use int.toUnicode()?